requirements-dev.txt

```diff
@@ -1,3 +1,2 @@
 -r requirements-test.txt
 django-debug-toolbar >=2.0, <2.1
-psycopg2==2.8.6
```

Comment:
why is this removed in requirements dev?

Reply:
It is moved to  requirements-test.txt.

Reply:
I see that but don't we need a Postgresql driver for development as well?

Reply:
Yep. But due to line 1, this is ensured.